packages/artifacts/chunks/languages/library-types.ts.txt

```diff
@@ -1,19 +1,16 @@
 // Warning: this file is generated!
 // Modifying it by hand it useless at best, and sabotage at worst.
 
-
 /*
  * language's metadata:
  *     name:    library
  *     version: 1
  */
 
-
 import {DynamicNode} from "@lionweb/core";
 
-
 export type Book = DynamicNode & {
-    classifier: "Book";
+    // classifier -> Book
```

Comment:
Commented on purpose?

Reply:
Yes: `DynamicNode.classifier` has type `Classifier`, so there's no point trying to use discriminated unions here.